src/Components/EnvironmentPicker/EnvironmentPicker.tsx

```diff
@@ -405,7 +405,7 @@ const EnvironmentPicker = ({
         resources: Array<IAzureResource | string>
```

Comment:
It says the containers are loaded but I only see a single result and I know there are many more in that storage account.
![image](https://user-images.githubusercontent.com/57726991/186233346-ac2e1358-b615-41fe-b2c0-58a1f99fde8c.png)
![image](https://user-images.githubusercontent.com/57726991/186233736-71609ce9-0a41-41d1-9032-1f773ef170ff.png)

Reply:
yes, this is expected behavior. you should wait until the storage account resource picker one above finishes fetching the data, until then we only show the passed values from props/local storage if exists. This was a requirement from @darsney, originally I was disabling the containers ResourcePicker (3rd combobox) until the storage container finishes fetching the data as I need the resource if of the storage account to fetch containers.

Reply:
even once it finished loading I never saw a list of containers. Also, it showed a loading spinner for the container picker which implies that it has finished loading the data for this picker. :/ That's pretty confusing.

Reply:
![image](https://user-images.githubusercontent.com/57726991/186544381-afddd151-36aa-432c-8a2c-e8816495540e.png)